R/viz.R

```diff
@@ -2942,48 +2947,48 @@ fancy_svbar = function(this_sample,
 fancy_cnbar = function(this_sample,
                        plot_subtitle = "n CNV Segments (barplots, left y-axis), n Affected bases for each CN state",
                        chr_select = paste0("chr", c(1:22)),
+                       cutoff = 160,
```

Comment:
is this a typo or intended to be 160 for the CN state?

Reply:
Oh, I just put a really high number as a placeholder and forgot to change it back to something more realistic.